### PR TITLE
Document the __toString() and export() related changes in ext/reflection

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/reflection/reflector.xml [hunk 1]
```diff
@@ -36,10 +36,6 @@
       <classname>Stringable</classname>
      </ooclass>
     </classsynopsisinfo>
-<!-- }}} -->
-    
-    <classsynopsisinfo role="comment">&Methods;</classsynopsisinfo>
```
Comment (kocsismate) [inline reference/reflection/reflector.xml:41]: Reflector doesn't have methods on its own: the `toString` method is inherited from `Stringable` (and we should set up a redirection), the export method has already been removed, so I linked in the "see also" section.

File: reference/reflection/reflector.xml [hunk 2]
```diff
@@ -64,16 +60,30 @@
...
        <entry>
-        <classname>Reflector</classname> implements
+        <methodname>Reflector::export</methodname> was removed.
+       </entry>
+      </row>
+      <row>
+       <entry>8.0.0</entry>
+       <entry>
+        <classname>Reflector</classname> extends
         <interfacename>Stringable</interfacename> now.
```
Comment (cmb69) [inline reference/reflection/reflector.xml:70]: I think we should tell users that `::__toString()` was previously declared on `Reflector`. Other than that, removing to `Reflector::__toString()` page makes sense to me, given that there was no real documentation.

**Review by cmb69 — COMMENTED** [review]
Thank you! Besides the commented issue, this looks good to me.

**Review by cmb69 — APPROVED** [review]
> WDYT about my recent commit?

Perfect! :)

**Pull request merged**